static/retirement/aggregatedsearch/index.md

```diff
@@ -11,7 +11,7 @@ In line with our [approach to the retirement of API endpoints](../), it has been
 - strategic technology direction is elsewhere
 - cost of upkeep is too high with similar services available
 
-The UI for these pages has already been moved to use our Search API and it is the intention of ONS to separate data and UI endpoints by domain in future. Data in JSON format will be accessible via our API found at <https://api.beta.ons.gov.uk/v1/search>. You can also browse the [full API Swagger specification](../../search/search/).
+The UI for these pages has already been moved to use our Search API, and it is the intention of ONS to separate data and UI endpoints by domain in the future. Data in JSON format will be accessible via our API found at <https://api.beta.ons.gov.uk/v1/search>. You can also browse the [full API Swagger specification](../../search/search/).
```

Comment:
I actually disagree with the additional 'the' in this sentence, but not enough to make you change it :D